prov/sockets/src/sock_ep_msg.c

```diff
@@ -379,440 +378,196 @@ static int sock_ep_cm_getpeer(struct fid_ep *ep, void *addr, size_t *addrlen)
 	return (len == sizeof(struct sockaddr_in)) ? 0 : -FI_ETOOSMALL;
 }
 
-static int sock_ep_cm_create_socket(void)
+static inline int sock_cm_send(int fd, const void *buf, int len)
```

Comment:
Please drop the inline (compiler will ignore it anyway) and let the compiler decide whether inlining makes sense or not.  Same comment applies to functions below.